tests/integration/ovs_test.py

```diff
@@ -15,106 +14,102 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
-#
 
 import pytest
-import yaml
 
-import libnmstate
+from libnmstate.schema import Interface
 from libnmstate.schema import InterfaceIPv4
+from libnmstate.schema import OVSBridge
+from libnmstate.schema import OVSBridgePortType
 from libnmstate.error import NmstateLibnmError
 
-from .testlib import statelib
-from .testlib.statelib import INTERFACES
+from .testlib import assertlib
+from .testlib.ovslib import add_port_to_bridge
+from .testlib.ovslib import create_bridge_subtree_state
+from .testlib.ovslib import ovs_bridge
+from .testlib.ovslib import ovs_interface
 
 
-OVS_BRIDGE_YAML_BASE = """
-interfaces:
-  - name: ovs-br0
-    type: ovs-bridge
-    state: up
-    bridge:
-      options:
-        fail-mode: ''
-        mcast-snooping-enable: false
-        rstp: false
-        stp: true
-"""
+@pytest.mark.xfail(
+    raises=NmstateLibnmError, reason='https://bugzilla.redhat.com/1724901'
+)
+def test_create_and_remove_ovs_bridge_with_min_desired_state():
+    with ovs_bridge('br1') as bridge_state:
```

Comment:
Please make ‘br1’ a constant, it is used a lot.

Reply:
You can use `TEST_BRIDGE1` or `BRIDGE1`, I do not mind. When you use it in multiple places, especially in a single test, it is preferred to use a constant in order to avoid mistakes.

Reply:
Done